include/openmc/plot.h

```diff
@@ -66,15 +89,26 @@ enum class PlotBasis {
 };
 
 enum class PlotColorBy {
-  cells = 1,
-  mats = 2
+  cells = 0,
+  mats = 1
 };
 
 //===============================================================================
 // Plot class
 //===============================================================================
-struct PlotBase {
+class PlotBase
+{
+  // Methods
+ public:
+  IdData get_id_map() const;
+  PropertyData get_property_map() const;
+
+ private:
+  template<class D>
+    D generate_data() const;
```

Comment:
Is there a reason you chose `D` as opposed to the more common `T`?

Reply:
No particular attachment to that. I've just always found it easier to name the template parameter something that will remind one of the data structure intended to replace it.